### ISSUE TITLE
Why is multi-task learning one head?

### ISSUE THREAD

**Issue by saeu5407** [HSE-asavchenko/face-emotion-recognition#31]
Thank you for your wonderful code sharing.

I'm looking at a multi-task learning code `train_emotion-pytorch.ipynb`. 

I know why you used FER, Valence, and Arousal in one head without dividing the head.

Thank you.

**Comment by HSE-asavchenko** [comment]
Hello!
Thanks for your question and sorry for delay in answering it.
It is just an engineering trick. It was easier for me to extend the EfficientNet model from timm package by using only one output layer with 10 units (8 facial expressions and Valence-Arousal). In such case, it is not necessary to extend the model and implement custom forward() pass. It is enough to create custom loss function and process the outputs of the model t evaluation and inference steps.
